src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/AbstractReferenceFinder.cs

```diff
@@ -243,17 +245,26 @@ protected static bool IdentifiersMatch(ISyntaxFactsService syntaxFacts, string n
 
                         var location = token.GetLocation();
                         var symbolUsageInfo = GetSymbolUsageInfo(token.Parent, semanticModel, syntaxFacts, semanticFacts, cancellationToken);
+
                         var isWrittenTo = symbolUsageInfo.IsWrittenTo();
                         locations.Add(new FinderLocation(token.Parent, new ReferenceLocation(
                             document, alias, location, isImplicit: false,
-                            symbolUsageInfo, candidateReason: match.reason)));
+                            symbolUsageInfo, GetAdditionalProperties(token.Parent, semanticModel, syntaxFacts), candidateReason: match.reason)));
                     }
                 }
             }
 
             return locations.ToImmutableAndFree();
         }
 
+        private static ImmutableArray<AdditionalProperty> GetAdditionalProperties(SyntaxNode node, SemanticModel semanticModel, ISyntaxFactsService syntaxFacts)
```

Comment:
Can you make this a protected static method that can also be called from `PropertyReferenceFinder`, instead of having duplicate private static methods?

Reply:
We discussed offline using an extension method instead of protected method.